src/libfuncs/enum.rs

```diff
@@ -359,33 +292,9 @@ pub fn build_match<'ctx, 'this>(
                         ))
                         .result(0)?
                         .into();
-                    let stack_ptr = helper
-                        .init_block()
-                        .append_operation(llvm::alloca(
-                            context,
-                            k1,
-                            llvm::r#type::opaque_pointer(context),
-                            location,
-                            AllocaOptions::new()
-                                .align(Some(IntegerAttribute::new(
-                                    IntegerType::new(context, 64).into(),
-                                    payload_layout.align() as i64,
-                                )))
-                                .elem_type(Some(TypeAttribute::new(payload_ty))),
-                        ))
-                        .result(0)?
-                        .into();
+                    let stack_ptr = helper.init_block().alloca1(context, location, payload_layout, Some(payload_layout.align()))?;
```

Comment:
i think the first payload_layout here should be payload_ty

Reply:
oh, okay, changed payload_layout to payload_ty